editor/editor_node.h

```diff
@@ -147,97 +147,87 @@ class EditorNode : public Node {
 	friend class SurfaceUpgradeTool;
 
 	enum MenuOptions {
+		// Scene menu.
 		FILE_NEW_SCENE,
 		FILE_NEW_INHERITED_SCENE,
 		FILE_OPEN_SCENE,
+		FILE_OPEN_PREV,
+		FILE_OPEN_RECENT,
 		FILE_SAVE_SCENE,
-		FILE_SAVE_SCENE_SILENTLY,
 		FILE_SAVE_AS_SCENE,
 		FILE_SAVE_ALL_SCENES,
-		FILE_SAVE_AND_RUN,
-		FILE_SAVE_AND_RUN_MAIN_SCENE,
-		FILE_RUN_SCENE,
-		FILE_SHOW_IN_FILESYSTEM,
-		FILE_EXPORT_PROJECT,
-		FILE_EXPORT_MESH_LIBRARY,
-		FILE_INSTALL_ANDROID_SOURCE,
-		FILE_EXPLORE_ANDROID_BUILD_TEMPLATES,
-		FILE_SAVE_OPTIMIZED,
-		FILE_OPEN_RECENT,
-		FILE_OPEN_OLD_SCENE,
 		FILE_QUICK_OPEN,
 		FILE_QUICK_OPEN_SCENE,
 		FILE_QUICK_OPEN_SCRIPT,
-		FILE_OPEN_PREV,
+		FILE_UNDO,
+		FILE_REDO,
```

Comment:
Renamed these 2 for consistency, but it does not look good. Though alternative is that they are the only 2 EDIT options.

Reply:
I think we can keep them as `EDIT_UNDO` / `EDIT_REDO`, that's clearer.